proto/beacon/p2p/v1/BUILD.bazel

```diff
@@ -26,8 +26,6 @@ ssz_gen_marshal(
     ],
     objs = [
         "BeaconBlocksByRangeRequest",
-        "BeaconBlocksByRootRequest",
```

Comment:
Delete `BeaconBlocksByRootRequest` in `messages.proto`?

Reply:
Its been deleted form before, this is just cleanup